src/virtualenv/create/creator.py

```diff
@@ -154,6 +154,15 @@ def run(self):
             safe_delete(self.dest)
         self.create()
         self.set_pyenv_cfg()
+        self._ensure_gitignore()
+
+    def _ensure_gitignore(self):
```

Comment:
let's not make it git specific, let's cover all VCS covered here https://www.python.org/dev/peps/pep-0610/#registered-vcs

a better name is: ``generate_vcs_ignore``, no need to make it private